Upstream: No, disables parts of test suite
Source: Michael Thomas <aelmalinka@gmail.com>
Reason: test/simple/test-net-server-bind specifically tests not passing a host, which defaults to 0.0.0.0

diff --git a/test/simple/test-net-server-bind.js b/test/simple/test-net-server-bind.js
index 56d3382..b72f47c 100644
--- a/test/simple/test-net-server-bind.js
+++ b/test/simple/test-net-server-bind.js
@@ -74,22 +74,9 @@ server3.listen(common.PORT + 2, '127.0.0.1', 127, function() {
 });
 
 
-// Backlog argument without host argument
-
-var address4;
-var server4 = net.createServer(function(socket) { });
-
-server4.listen(common.PORT + 3, 127, function() {
-  address4 = server4.address();
-  console.log('address4 %j', address4);
-  server4.close();
-});
-
-
 process.on('exit', function() {
   assert.ok(address0.port > 100);
   assert.equal(common.PORT, address1.port);
   assert.equal(common.PORT + 1, address2.port);
   assert.equal(common.PORT + 2, address3.port);
-  assert.equal(common.PORT + 3, address4.port);
 });
